src/h5web/visualizations/shared/ButtonGroup.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+interface Button {
+  label: string;
+  isSelected: boolean;
+  onClick: () => void;
+}
+
+interface Props {
+  buttons: Button[];
+  className?: string;
+  buttonClassName?: string;
+}
+
+function ButtonGroup(props: Props): JSX.Element {
+  const { buttons, className, buttonClassName } = props;
+
+  if (buttons.length === 0) {
+    return <></>;
+  }
+
+  return (
+    <div role="tablist" className={className}>
```

Comment:
This is not quite a `tablist`, but more a `radiogroup`. See the text alignment options in this toolbar example: https://www.w3.org/TR/wai-aria-practices/examples/toolbar/toolbar.html

This will fix the tests.